tests/test__plotting.py

```diff
@@ -11,6 +11,14 @@ def test__get_pyplot():
         with pytest.raises(ImportError):
             test_module._get_pyplot()
 
+    import matplotlib
+
+    # Set a backend different to MacOSX (default on Mac OS) to avoid the error
+    #   RuntimeError: Python is not installed as a framework
+    # using Python 2.7 in a virtualenv on Mac OS.
+    # See also https://matplotlib.org/faq/osx_framework.html.
+    matplotlib.use('pdf')
```

Comment:
I would more into :
```suggestion
    if sys.platform == 'darwin' and six.PY2:
        matplotlib.use('pdf')
```
so we can test the default one on linux ?

Reply:
Yes I think it's fine, we can change the matplotlib backend only in the known problematic case.